public/javascripts/now.js

```diff
@@ -670,6 +672,46 @@ function updateWeather(coords, iconMap) {
   }, 600000);
 }
 
+/* Date and time functions */
+
+function initDatetime() {
+  updateDatetime();
+}
+
+function updateDatetime() {
+  $('#datetime #date').text(getCurrentDate());
+  $('#datetime #time').text(getCurrentTime());
+
+  setTimeout(function() {
+    updateDatetime();
+  }, 3000);
```

Comment:
Might as well change the timeout to 1000. It shouldn't make a difference performance wise, and we'll have a slightly more accurate time. Also consider using [`setInterval`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval) (I know the existing code isn't the prettiest haha).